code.md

```diff
@@ -4,4 +4,6 @@
 - https://github.com/howdyai/botkit-cms : An open tool for designing, building and managing interactive dialog systems
```

Comment:
Hi @Jayita10 

The task is to add a header to the markdown file as below:

#Code

Thanks

Reply:
Sorry I didn't understand. Please Check now

Reply:
perfect! Thanks!
I see that you are into web development. Please have a look at https://github.com/dermatologist/fhir-questionnaire-render-react when you have time!